examples/generic-azure.yaml

```diff
@@ -166,6 +166,19 @@ groups:
           ## Remediation
           Adjust the corosync transport parameter as recommended on the Azure best practices.
 
+          ## References
+          - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
+        scored: true
+      - id: 1.1.6.runtime
+        description: "Corosync transport parameter is set to udpu (runtime)"
+        audit: 'corosync-cmapctl | grep "totem.transport (str) = " | sed "s/.*= //"'
+        tests:
+          test_items:
+            - flag: "udpu"
+        remediation: |
+          ## Remediation
+          Adjust the corosync transport parameter as recommended on the Azure best practices.
```

Comment:
I think this kind of remediation are not really helpful.
We should explain here what the user should do, and later on reference them to the main documentation page.

At least, personally, and even knowing our cluster, I cannot really get any meaning from this message.

Shouldn't we have something like `Adjust corosync transport parameter to udpu. This is configured in /etc/corosync/corosync, in the totem.transport field`. Or something similar.

Reply:
> I think this kind of remediation are not really helpful.
> We should explain here what the user should do, and later on reference them to the main documentation page.

I totally agree with this. 

> Adjust corosync transport parameter to udpu. This is configured in /etc/corosync/corosync, in the totem.transport field

It's still not f a big use. I will amend it and explain in detail how to fix it.